jujugui/static/gui/src/app/components/profile/bundle-list/bundle-list.js

```diff
@@ -184,49 +129,25 @@ class ProfileBundleList extends React.Component {
             columnSize: 1
           }],
           expandedContent: (
-            <div className="profile-bundle-list__expanded">
-              <div className="eight-col profile-bundle-list__expanded-leading">
-                {bundle.name}
-              </div>
-              <div className="two-col profile-bundle-list__expanded-leading">
-                {bundle.machineCount}
-              </div>
-              <div className="one-col profile-bundle-list__expanded-leading">
-                {bundle.unitCount}
-              </div>
-              <div className="one-col last-col profile-bundle-list__expanded-leading">
-                {version}
-              </div>
-              <div className="seven-col">
-                <p>{bundle.description}</p>
-                <EntityContentDiagram
-                  diagramUrl={this.props.charmstore.getDiagramURL(bundle.id)} />
-              </div>
-              <div className="five-col last-col">
-                <div>
-                  <a href={bundle.bugUrl}
-                    onClick={this._stopPropagation.bind(this)}
-                    target="_blank">
-                    Bugs
-                  </a>
-                </div>
+            <ProfileExpandedContent
+              changeState={this.props.changeState}
+              entity={bundle}
+              getDiagramURL={this.props.charmstore.getDiagramURL}
```

Comment:
Do we need to bind this? Just asking for double checking.

Reply:
Thanks, it probably should be.